gatling-ss7/src/main/scala/io/snice/gatling/ss7/engine/Ss7Client.scala

```diff
@@ -63,29 +66,28 @@ class Ss7Client(mapStack: MAPStack,
     mp
   }
 
+  class RequestId(val transactionId: Long, val invokeId: Long) {
+    def canEqual(other: Any): Boolean = other.isInstanceOf[RequestId]
+
+    override def equals(other: Any): Boolean = other match {
+      case that: RequestId =>
+        (that canEqual this) &&
+          transactionId == that.transactionId &&
+          invokeId == that.invokeId
+      case _ => false
+    }
+
+    override def hashCode(): Int = {
```

Comment:
What do you use it for?

Reply:
The map uses the hash code of the key to compute the index within the table to get its mapped value